linetools/analysis/absline.py

```diff
@@ -69,3 +73,74 @@ def aodm(spec,idata):
 
     # Return
     return ntot, np.sqrt(tvar), flg_sat
+
+
+def log_clm(obj):
+    """Return logN and sig_logN given linear N, sigN
+    Parameters
+    ---------
```

Comment:
Also dashes must be the same length as the heading for Sphinx (Sorry - this is a bit annoying, I know)